# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/coreutils/cross.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./lib/putenv.c.orig	2004-10-01 05:45:46.844122664 +0200
+++ ./lib/putenv.c	2004-10-01 05:45:56.619636560 +0200
@@ -37,8 +37,10 @@
    with a non-const argument.  That would conflict with the declaration of
    rpl_putenv below (due to the #define putenv rpl_putenv from config.h).  */
 
+#if 0
 void *malloc ();
 void free ();
+#endif
 
 #include <string.h>
 
--- ./lib/getline.c.orig	2004-10-01 05:46:07.911919872 +0200
+++ ./lib/getline.c	2004-10-01 05:46:56.900472480 +0200
@@ -25,8 +25,10 @@
 #include "getdelim.h"
 #include "getline.h"
 
+#if 0
 ssize_t
 getline (char **lineptr, size_t *n, FILE *stream)
 {
   return getdelim (lineptr, n, '\n', stream);
 }
+#endif
